config/routes.rb

```diff
@@ -39,7 +39,10 @@
       post :saml_slo_callback, to: 'sessions#saml_slo_callback'
     end
 
-    resource :user, only: [:show]
+    resource :user, only: [:show] do
+      get 'authorized/:policy/:policy_action', to: 'users#authorized_for_service?'
```

Comment:
So my main observation on this functionality is that we're making the FE responsible for understanding a new concept of a `policy` which it didn't need before.

I find the `BackendService` pattern a little unclear, tbh, but tying off of that as a key for checking additional authorization errors would be more self explanatory than introducing a separate name for a policy. I guess I'm arguing for a 1<>1 mapping with the name of the service being exposed by vets-api (eg exposing `profile` and internally mapping that to an EVSS authorization). 

I feel this would be more straightforward as we start to think of vets-api less as a single service, and more as a collection of services. We expose the "profile" service, which internally utilizes EVSS (and possibly simultaneously Vet360) - and every external reference to this API does not necessarily need to understand that EVSS is involved in the request.

Reply:
I agree that it’s not great that policy details are leaking up into into the client. The backend services list went from data needed to toggle navigation depending on what the user had authorization for to preemptively displaying an unauthorized message without making an additional api call. My original intent was to add auth details to a unauthorized api response but we need the details before such a call is made.

We might be able to move up one of the data collection calls, and if we add details to the unauth response we wouldn’t need this separate endpoint.